src/mlpack/methods/ann/layer/lookup_impl.hpp

```diff
@@ -37,15 +37,14 @@ void Lookup<InputDataType, OutputDataType>::Forward(
   const size_t seqLength = input.n_rows;
   const size_t batchSize = input.n_cols;
 
-  output.set_size(seqLength * embeddingSize, batchSize);
+  output.set_size(embeddingSize * seqLength, batchSize);
 
   for (size_t i = 0; i < batchSize; ++i)
   {
     //! ith column of output is a vectorized form of a matrix of shape
-    //! (seqLength, embeddingSize) selected as a combination of rows from the
-    //! weights. The MultiheadAttention class requires this particular ordering
-    //! of matrix dimensions.
-    output.col(i) = arma::vectorise(weights.rows(
+    //! (embeddingSize, seqLength) selected as a combination of rows from the
```

Comment:
```suggestion
    //! (embeddingSize, seqLength) selected as a combination of columns from the
```
By the way, you don't need the exclamation after `//` since Doxygen ignores comments that aren't attached to a function or class etc.